cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/type/PersonDrawWidget.kt

```diff
@@ -0,0 +1,121 @@
+package com.cognifide.cogboard.widget.type
+
+import com.cognifide.cogboard.CogboardConstants
+import com.cognifide.cogboard.config.service.BoardsConfigService
+import com.cognifide.cogboard.storage.ContentRepository
+import com.cognifide.cogboard.widget.BaseWidget
+import io.vertx.core.Vertx
+import io.vertx.core.json.JsonArray
+import io.vertx.core.json.JsonObject
+import java.time.Instant
+import java.time.LocalDateTime
+import java.time.ZoneId
+import java.time.ZoneOffset
+import kotlin.random.Random
+
+class PersonDrawWidget(vertx: Vertx, config: JsonObject, boardService: BoardsConfigService = BoardsConfigService()) :
```

Comment:
Logic in this class is quite complex - it would be great to have JUnit tests for BL.
I thing you could extract BL to seperate class and test it there so you don't have to mock a lot

Reply:
Yes. I agree with Szymon in 100%, A lot of logic here which can be quite easy to test. Bu you need to have the logic in sepeare classs(es).